app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -836,10 +836,10 @@ private void passInfoToSheet(Place place) {
             updateMarker(isBookmarked, this.place);
         });
 
-        wikipediaButton.setEnabled(place.hasWikipediaLink());
+        wikipediaButton.setVisibility(place.hasWikipediaLink()==true?View.VISIBLE:View.GONE);
```

Comment:
If `place.hasWikipediaLink()?View.VISIBLE:View.GONE` works, it would be [better](https://softwareengineering.stackexchange.com/questions/12807/make-a-big-deal-out-of-true) :-)

Reply:
OK I'll do that